### ISSUE TITLE
Error building blocks with MEV Boost & Web3Signer

### ISSUE THREAD

**Issue by melenxyz** [status-im/nim-beacon-chain#4382]
**Describe the bug**
Nimbus-CL was unable to build a block using MEV Boost & Web3Signer.

**To Reproduce**
1. Platform details (OS, architecture): 
DappNode: 
• Intel NUC i7 
• 32 GBDDR4 RAM 
• 4TB NVMe

2. Branch/commit used: 
Dappnode package Version: 1.0.5, corresponding to Nimbus version v22.10.1 [/ipfs/QmTbWRKVQA1aXqNzVHbgVfUSSAtSzBdmWM1meZFQryN1iJ]
https://github.com/dappnode/DAppNodePackage-nimbus/releases/tag/v1.0.5

3. Commands being executed: 
MEV Boost 1.14 was enabled, via the DappNode script itself, and MEV Boost was configured with "-min-bid 0.05" which is possibly unsupported by Nimbus?

4. Relevant log lines: 

Web3Signer:
```
vert.x-eventloop-thread-0 | ERROR | LogErrorHandler | Failed request: http://web3signer.web3signer.dappnode/api/v1/eth2/sign/[HEX]

io.vertx.ext.web.validation.BodyProcessorException: [Bad Request] Validation error for body application/json: No schema matches
```

Nimbus:
```
proposeBlockMEV: getBlindedBeaconBlock failed topics="beacval" slot=SLOT head=HEAD validator_index=INDEX blindedBlock="err(Unable to sign block: Remote signer returns 400 Bad Request Format Error)" error="Unable to sign block: Remote signer returns 400 Bad Request Format Error"

Slashing protection activated for block proposal topics="beacval" blockRoot=ROOT blck="(slot: SLOT, proposer_index: INDEX, parent_root: \"ROOT\", state_root: \"ROOT\", eth1data: (deposit_root: ROOT, deposit_count: COUNT, block_hash: HASH), graffiti: \"validating_from_DAppNode\", proposer_slashings_len: 0, attester_slashings_len: 0, attestations_len: 128, deposits_len: 0, voluntary_exits_len: 0, sync_committee_participants: 497)" signingRoot=ROOT validator=VAL slot=SLOT existingProposal="(kind: DoubleProposal, existingBlock: BLOCK)"
```

**Additional context**
Erigon is used as an Execution Layer, the node itself was otherwise performing attestations with no issues, the Nimbus logs prior to that error showed no signs of malfunction. 
I included logs from Nimbus and Web3Signer but didn't get a chance to grab MEVBoost's ones

**Comment by jlr13** [comment]
I‘ve had exactly the same problem leading to a missed block proposal. Also running on dappnode but with Nethermind as execution client.

**Comment by arnetheduck** [comment]
This seems to be related to the combination of running an external block builder and web3signer - we're looking into it but until then, the recommendation would be to turn off external block building with `--payload-builder=false`: https://nimbus.guide/external-block-builder.html

**Comment by melenxyz** [comment]
Thanks a lot @arnetheduck 
Also note that I had a -min-bid setup which may or may not have created further issues. 
Regardless, let me know if I can provide any details to help on the matter

**Comment by jlr13** [comment]
In my case it happened without the -min-bid set, thanks @arnetheduck for taking a look at this!

**Comment by arnetheduck** [comment]
We're testing a fix that will hopefully resolve this and similar issues with web3signer: https://github.com/status-im/nimbus-eth2/pull/4407

**Comment by zah** [comment]
The issue was fixed in the linked PR above, which was shipped in Nimbus 22.11.1
